pkg/rtc/mediatrack.go

```diff
@@ -631,6 +631,9 @@ func (t *MediaTrack) handlePublisherFeedback(packets []rtcp.Packet) {
 	var totalLost uint32
 	var maxSeqNum uint32
 
+	//forward to telemetry
+	t.params.Telemetry.HandleRTCP(livekit.StreamType_UPSTREAM, t.params.ParticipantID, t.ID(), packets)
```

Comment:
Does this include padding packets? Not sure if it changed, but this was giving me a lower packet count and byte count than the interceptor

Reply:
This is not used for packet count and byte count. They are received from buffer.Buffer as before. RTCP values are same as from interceptor, I have checked that.